packages/aragon-wrapper/src/rpc/handlers/addressIdentity.js

```diff
@@ -0,0 +1,14 @@
+export default function (request, proxy, wrapper) {
```

Comment:
Not directly related, but I see that some of the handlers are async and some aren't. I guess it doesn't matter as long as they return a promise (https://github.com/aragon/aragon.js/blob/master/packages/aragon-wrapper/src/rpc/handlers/index.js#L28). 

Am I missing something?

Reply:
`from()` kind of abstracts this all away, so that any value (synchronous, asynchronous, observable) should be technically handled correctly. `ObservableInput` (the input type for `from()`) is a really wide net.